svthalia/concrexit
# Optimize many views

Closes #1617 



### Summary
Fixes a lot of O(n) query problems. I still want to look into photos (the album overview) because my code there worked on the page but seemed to break the test.

I also want to check the API for such problems. Admin and website should mostly be fine now.

### How to test
Steps to test the changes you made:
1. Go to any page
2. Observe that it works correctly
3. See in DJDT that there are significantly less queries than before
